openwisp_monitoring/device/apps.py

```diff
@@ -25,7 +27,7 @@ def ready(self):
         self.device_recovery_detection()
 
     def connect_device_signals(self):
-        from openwisp_controller.config.models import Device
+        Device = load_model('config', 'Device')
```

Comment:
I think we can move the following definitions to the top of this file:

```
        Device = load_model('config', 'Device')
        DeviceData = load_model('device_monitoring', 'DeviceData')
        DeviceMonitoring = load_model('device_monitoring', 'DeviceMonitoring')
```

Reply:
unfortunately it can't be done because the models are not loaded at that point yet, trying to load them will raise an exception